### DISCUSSION TITLE
SEB 2.3 for macOS and microphone access

### DISCUSSION THREAD

**Discussion by ArtemKislicyn** [SafeExamBrowser/seb-mac#104] [Q&A]
Hi!

There was a problem: in the latest version of SEB for macOS, the microphone does not connect. Settings were set to allow access to the microphone, we tried to change the userAgent, but access nevertheless cannot be allowed.

In the console, you can see:

`userMedia.js enumiratedDevices in unssupported on this browser`

How can I allow access? Access is allowed in WebRTC, the microphone is working. It was checked from several devices.

**Comment by danschlet** [comment]
That is being discussed in issue #66.
SEB 3.0 for macOS which allows microphone access on compatible macOS versions will be released soon.
